app/graphql/schema.graphql

```diff
@@ -1,3 +1,48 @@
+"""
+Autogenerated input type of AttachFilesToSample
+"""
+input AttachFilesToSampleInput {
+  """
+  A unique identifier for the client performing the mutation.
+  """
+  clientMutationId: String
+
+  """
+  A list of files (signedBlobId) to attach to the sample
+  """
+  files: [String!]!
+
+  """
+  The Node ID of the sample to be updated.
+  """
+  sampleId: ID!
+}
+
+"""
+Autogenerated return type of AttachFilesToSample.
+"""
+type AttachFilesToSamplePayload {
+  """
+  A unique identifier for the client performing the mutation.
+  """
+  clientMutationId: String
+
+  """
+  A list of errors that prevented the mutation.
+  """
+  errors: [String!]!
```

Comment:
Should this not be `JSON!` since the errors is a `JSON` type?

Reply:
Yeah he just forgot to run the rake task to regenerate the graphql schema after updating it which is also why the tests are failing.

Reply:
Ah makes sense!

Reply:
the schema has been updated